redis_enterprise/metadata.csv

```diff
@@ -0,0 +1,180 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric,
+rdse.bdb_avg_latency,rate,15s,second,operations/sec,Average latency of operations on the DB (seconds); returned only when there is traffic,-1,redis_enterprise_database,bdb_avg_latency,,
```

Comment:
The unit is second, the sub-unit, or per-unit-name, is operations/sec. This is what I was told regarding sub-units

On Thu, Feb 22, 2024 at 2:21 PM Emily Marshall <[emily.marshall@datadoghq.com](mailto:emily.marshall@datadoghq.com)> wrote:
Hi John,

I believe based off your use case it would be the following:
unit_name = bytes
per_unit_name = per second 

I don't believe there is a required way/validation on what you include for the sub unit so you could include expiration if you don't think that would confuse customers. Per unit names are not a requirement, but optional. 

Cheers,
Emily